test/NodeGuard.Tests/Rpc/NodeGuardServiceTests.cs

```diff
@@ -1104,7 +1104,7 @@ public async Task GetNodes_RequestIncludeUnmanaged_ReturnsUnmanagedNodes()
             };
 
             nodeRepositoryMock.Setup(repo => repo.GetAll()).ReturnsAsync(sampleNodes);
-            nodeRepositoryMock.Setup(repo => repo.GetAllManagedByNodeGuard()).ReturnsAsync(new List<Node>());
+            nodeRepositoryMock.Setup(repo => repo.GetAllManagedByNodeGuard(true)).ReturnsAsync(new List<Node>());
```

Comment:
You don't need to modify the tests, because you already added a default in the parameter
```
public Task<List<Node>> GetAllManagedByNodeGuard(bool includeDisabledNodes = true)
```

Reply:
It would be nice to create a test case for this. I know is a simple thing, but it's nice to have.

Reply:
might be because of the mock. Leave it like it was